website/docs/reference/resource-configs/tags.md

```diff
@@ -164,10 +200,65 @@ seeds:
 
 </File>
 
+### Apply tags to saved queries
+
+<VersionBlock lastVersion="1.8">
+
+Applying tags to saved queries is only available in dbt Core versions 1.9 and later.
+
+</VersionBlock>
+
+<VersionBlock firstVersion="1.9">
+
+This following example shows how to apply a tag to a saved query in the `dbt_project.yml` file. The saved query is then tagged with `order_metrics`.
+
+<File name='dbt_project.yml'>
+
+```yml
+[saved-queries](/docs/build/saved-queries):
+  jaffle_shop:
+    customer_order_metrics:
+      +tags: order_metrics
+```
+
+</File>
+
+Then, run part of your project like so:
```

Comment:
Hiya @mirnawong1 

Apologies for the silly question - which part of the project would the user need to run? (ie the first part, last etc)? And does it matter which part of their project they need to run?

Kind Regards
Natalie

Reply:
oo i'll clarify that they can run this command  to find the resources with a specific tag

Reply:
changed it to this, good shout @nataliefiann !
![Screenshot 2024-12-24 at 12 21 16](https://github.com/user-attachments/assets/b9aea1c2-e087-4667-aaa4-5025b84ce7db)